english/necklace/necklace.md

```diff
@@ -2,9 +2,6 @@ Remove 25 marshmallows from the wall,
 Create 5 sets of 5 marshmallows each for a total of 25 in all,
 Stick 4 marshmallows together from each set together like a string,
 Separate each set with another marshmallow half an inch in between,
-For sure you will enjoy the look of each bead,
-Marshmallow here I come I wil eat you in peace.
+For sure you will enjoy the look of each bead.
 
-After making about 3 necklaces,
-I realized this is will be the next big hit of the 22nd century,
-so I decided to make a startup marketing to those individuals who are into fashion as well as having a sweet tooth.
+After making about 3 necklaces, I realized this is will be the next big hit of the 22nd century, so I decided to make a startup marketing to those individuals who are into fashion as well as having a sweet tooth.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.